deploy-templates/values.yaml

```diff
@@ -460,7 +460,7 @@ edp-tekton:
 # - https://github.com/epam/edp-cluster-add-ons
 # - https://github.com/epam/edp-keycloak-operator
 sso:
-  # -- Install OAuth2-proxy and Keycloak CRs as a part of KubeRocketCI deployment.
+  # -- Install Keycloak CRs as a part of KubeRocketCI deployment.
```

Comment:
@NikolayMarusenko please have a look, do we still need all these kk realm and brokers from edp-install?

Reply:
we also have gerrit-operator and we can migrate resources creation from edp-install to gerrit-operator. @alezander86 cc